R/par.R

```diff
@@ -186,10 +190,17 @@ igraph_options <- function(...) {
 
   ## Set them
   names(temp) <- paste0("igraph::", names(temp))
-  do.call(set_config_in, c(temp, list(.in = parent.frame())))
+  do.call(set_config_in, c(temp, list(.in = .in)))
   invisible(old)
 }
 
+local_igraph_options <- function(..., .in = parent.frame()) {
+  old <- igraph_options(..., .in = .in)
+  print(old)
```

Comment:
Is this `print()` statement necessary here or is it only for debugging purposes?

Reply:
Good catch, removed.